normandy/selfrepair/static/js/self_repair_runner.js

```diff
@@ -114,12 +119,16 @@ function runRecipe(recipe) {
  * Generate a context object for JEXL filter expressions.
  * @return {object}
  */
-function filterContext() {
-    return {
-        normandy: classifyClient(),
-        telemetry: {},
-        events: {},
-    }
+export function filterContext() {
+    return new Promise(resolve => {
+        classifyClient().then(classifiedClient => {
+            resolve({
+                normandy: classifiedClient,
+                telemetry: {},
+                events: {},
+            })
+        })
+    })
```

Comment:
To me, having a promise inside a `new Promise` block seems like an anti-pattern. What do you think about this?

```js
return classifyClient()
.then(classifiedClient => {
  return {
    normandy: classifiedclient,
    telemetry: {},
    events: {},
  }
});
```

Besides this, I don't think we actually need `telemetry` and `events` on this object: Jexl will just return undefined if actions try to access them. That's even true if someone tries to access something nested, like `telemetry.general.user_id`.